.travis.yml

```diff
@@ -2,19 +2,14 @@ language: php
 sudo: false
 
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+  - 7.1
```

Comment:
I think if we're testing against 7.1, we'll want to test against 7.2 as well

Reply:
Yes, of course :)